lib/models/app/app_state.dart

```diff
@@ -16,6 +16,8 @@ import 'package:crowdleague/models/conversations/conversation/vm_conversation_pa
 import 'package:crowdleague/models/conversations/new_conversation/vm_new_conversation_page.dart';
```

Comment:
Not much change to app state, just the list of page data

Reply:
thanks :)